cmd/theatre-consoles/create.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/gocardless/theatre/pkg/workloads/console/runner"
+	consoleRunner "github.com/gocardless/theatre/pkg/workloads/console/runner"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	createSelector = create.Flag("selector", "Selector that matches console template").Required().String()
+	createTimeout  = create.Flag("timeout", "Timeout for the new console").Duration()
+	createReason   = create.Flag("reason", "Reason for creating console").String()
+	createCommand  = create.Arg("command", "Command to run in console").Strings()
+)
+
+func Create(ctx context.Context, logger kitlog.Logger, runner *runner.Runner) error {
+	var err error
+
+	// Create and attach to the console
+	tpl, err := runner.FindTemplateBySelector(metav1.NamespaceAll, *createSelector)
```

Comment:
I think that this only works if the user has permission to query ConsoleTemplates at the cluster scope, which not everyone/everything (service accounts) will have. I suspect we'll need to add namespace as well to get around this, but happy to leave that until a later PR.

Reply:
Actually yes, I think we definitely need to be able to supply namespace, in order to be able to address console templates which exist in multiple namespaces with the same label sets:

```console
$ go run ./... create --selector 'app=etcd'
consoles: error: unexpected error: expected to discover 1 console template, but actually found: [platform/etcd-cluster-default production/etcd-cluster-default staging/etcd-cluster-default]
```